modules/reactor/include/utils/HexagonalLatticeUtils.h

```diff
@@ -0,0 +1,823 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Moose.h"
+#include "MooseTypes.h"
+
+#ifndef CARDINAL_ENUM_H
+namespace channel_type
```

Comment:
If this header gets included before `CardinalEnum.h`, then there will still be trouble

Reply:
umm not much I can do about it though. Next time we update moose in cardinal we ll just have to consolidate the utils